docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -12,10 +12,11 @@ This hardening guide is intended to be used for RKE clusters and is associated w
 
 | Rancher Version | CIS Benchmark Version | Kubernetes Version           |
 |-----------------|-----------------------|------------------------------|
-| Rancher v2.7    | Benchmark v1.23       | Kubernetes v1.23 up to v1.25 |
+| Rancher v2.7    | Benchmark v1.7       | Kubernetes v1.24 up to v1.25 |
```

Comment:
As per @Priyashetty17 suggestion.
```suggestion
| Rancher v2.7    | Benchmark v1.7       | Kubernetes v1.25 |
```

Reply:
@andypitcher I did not realize this was the hardening guide. I got confused with the self assessment guide for v1.7. Since the upstream only supports cis v1.23 as of now, should we leave the CIS Benchmark Version in the Hardening Guide at 1.23 and K8s version "v1.23 up to v1.26"?

Reply:
This is a good point, we could have one hardening-guide page for each distro, with one tab page for each CIS version (v1.23, v1.24 and v1.7) then, since cis-1.23 is the only option available upstream (it means we need to develop/provide cis-1.24 and cis-1.7 upstream).
@macedogm do you have some recommendation ?

Reply:
@andypitcher I partially agree with the proposal:

> we could have one hardening-guide page for each distro

We already have this.

> with one tab page for each CIS version (v1.23, v1.24 and v1.7)

True. For the moment each tab is named as the K8s version, we would only have to change it to be named according to the CIS version too, perhaps from:

`v1.25 and Newer | v1.24 and Older`

To:

`CIS 1.7 (K8s v1.25) | CIS 1.24 (K8s v1.24) | CIS 1.23 / (K8s v1.22-v1.23)`

Is that what you mean?

![image](https://github.com/rancher/rancher-docs/assets/17654553/1e49bf13-3e15-4fbf-8d08-27e599881e48)

> then, since cis-1.23 is the only option available upstream (it means we need to develop/provide cis-1.24 and cis-1.7 upstream).

I didn't get this part. Can you please elaborate further?

Reply:
Thanks for the explanation. While the RKE2 flag isn't improved, should we just mention that users must use the `cis-1.23` for all CIS versions then?